dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/HttpStatAttributes.java

```diff
@@ -30,44 +37,100 @@ public class HttpStatAttributes {
 	 * by surrounding the the chainFilter with try catch. But we have no way
 	 * of capturing application exception of Jaxrs/restfulws exceptions
 	 */
-	Optional<Exception> exception = Optional.ofNullable(null);
-	Optional<String> errorType = Optional.ofNullable(null);
+	private final Exception exception;
+	private final String errorType;
 	
 	/*
 	 * Conditionally required as per HTTP Semantics Convention
 	 */ 
-	Optional<String> httpRoute = Optional.ofNullable(null); 
-	Optional<Integer> responseStatus = Optional.ofNullable(null); 
+	private final String httpRoute;
+	private final Integer responseStatus;
 	
 	
-	public boolean validate() {
-		return (requestMethod != null) &&
-				(scheme != null) &&
-				(networkProtocolName != null) &&
-				(networkProtocolVersion != null) &&
-				(serverName != null) &&
-				(serverPort != 0);
+	/**
+	 * Constructor for HttpStatsAttributes. This should not be called directly, but
+	 * should be instantiated through {@link Builder#build()}
+	 * 
+	 * @param builder see {@link Builder}
+	 * @throws IllegalStateException if the builder's validation fails
+	 */
+	public HttpStatAttributes(Builder builder) throws IllegalStateException {
```

Comment:
```suggestion
	HttpStatAttributes(Builder builder) throws IllegalStateException {
```
Doing so prevents it being called outside of this package.  Namely we want only the inner class to call it.